src/validationinterface.h

```diff
@@ -17,6 +17,8 @@
 
 /** Default limit on batch scanner memory usage in MiB. */
 static const size_t DEFAULT_BATCHSCANNERMEMLIMIT = 100;
```

Comment:
I think this can be removed. It’s only used to initialize `nBatchScannerMemLimit`, and the only use of that has been removed in this PR.